css/main.css

```diff
@@ -280,44 +465,54 @@ header .menu img {
   margin-top: 9rem;
 }
 
-.aboutme .description {
+.aboutme__description {
   color: var(--white);
   margin-top: 1rem;
+  line-height: 2.4rem;
 }
 
-.aboutme .aboutme_btn {
+.aboutme_btn {
   background-color: var(--orange);
   border: none;
   color: var(--white);
   padding: 1.5rem;
   margin-top: 3rem;
   font-size: 1.7rem;
```

Comment:
- [ ] Kindly add button interaction styles for **all** the buttons (you missed the cards buttons, the button in the about me section, and the form submission button) 💯